src/scripts/showNsfw.js

```diff
@@ -0,0 +1,27 @@
+import { getOptions } from './utils/jsTools.js';
+import { activeProject } from './utils/user.js';
+import { mutationManager } from './utils/mutation.js';
+
+let nsfwButtonQuery = 'article .co-18-plus + div button'
+
+export async function main () {
+  let {handles} = await getOptions('showNsfw')
+  handles = handles.split(',').map(h => h.replaceAll(/[@ \n]/g, ''))
+
+  const currentHandle = activeProject.handle
+
+  if (handles.includes(currentHandle)) {
+    mutationManager.start(nsfwButtonQuery, elems => {
```

Comment:
the `func` param of a mutation callback shouldn't be an anonymous function, because it's the function that serves as the key in the map, not the selector